web/src/app/schedules/calendar-subscribe/CalendarSubscribeForm.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react'
+import { PropTypes as p } from 'prop-types'
+import { FormContainer, FormField } from '../../forms'
+import { Grid, TextField } from '@material-ui/core'
+import { ScheduleSelect } from '../../selection'
+import MaterialSelect from '../../selection/MaterialSelect'
+import _ from 'lodash-es'
+
+export const reminderMinutesOptions = [
+  {
+    label: 'At time of shift',
+    value: '0',
```

Comment:
Just saw we are doing this in ep stuff too -- assuming it has to be that way for the control to function, mind adding a real short comment in both places explaining why?

Reply:
Looks like we are asking for a string on `value` for no (apparent) reason. I've changed it to `any` since a component using `MaterialSelect` should have free rein on the data they would like to set the value as, IMO.